core/src/main/java/de/danoeh/antennapod/core/feed/FeedItem.java

```diff
@@ -125,6 +126,37 @@ public FeedItem(long id, String title, String itemIdentifier, String link, Date
         this.hasChapters = hasChapters;
     }
 
+    public static FeedItem fromCursor(Cursor cursor) {
+        int indexId = cursor.getColumnIndex(PodDBAdapter.KEY_ID);
+        int indexTitle = cursor.getColumnIndex(PodDBAdapter.KEY_TITLE);
+        int indexLink = cursor.getColumnIndex(PodDBAdapter.KEY_LINK);
+        int indexPubDate = cursor.getColumnIndex(PodDBAdapter.KEY_PUBDATE);
+        int indexPaymentLink = cursor.getColumnIndex(PodDBAdapter.KEY_PAYMENT_LINK);
+        int indexFeedId = cursor.getColumnIndex(PodDBAdapter.KEY_FEED);
+        int indexFlattrStatus = cursor.getColumnIndex(PodDBAdapter.KEY_FLATTR_STATUS);
+        int indexHasChapters = cursor.getColumnIndex(PodDBAdapter.KEY_HAS_CHAPTERS);
+        int indexRead = cursor.getColumnIndex(PodDBAdapter.KEY_READ);
+        int indexItemIdentifier = cursor.getColumnIndex(PodDBAdapter.KEY_ITEM_IDENTIFIER);
+        int indexAutoDownload = cursor.getColumnIndex(PodDBAdapter.KEY_AUTO_DOWNLOAD);
+
+        long id = cursor.getInt(indexId);
+        assert(id > 0);
+        String title = cursor.getString(indexTitle);
+        String link = cursor.getString(indexLink);
+        Date pubDate = new Date(cursor.getLong(indexPubDate));
+        String paymentLink = cursor.getString(indexPaymentLink);
+        long feedId = cursor.getLong(indexFeedId);
+        boolean hasChapters = cursor.getInt(indexHasChapters) > 0;
+        FlattrStatus flattrStatus = new FlattrStatus(cursor.getLong(indexFlattrStatus));
+        int state = cursor.getInt(indexRead);
+        String itemIdentifier = cursor.getString(indexItemIdentifier);
+        boolean autoDownload = cursor.getInt(indexAutoDownload) > 0;
+
+        FeedItem item = new FeedItem(id, title, link, pubDate, paymentLink, feedId, flattrStatus,
+                hasChapters, null, state, itemIdentifier, autoDownload);
```

Comment:
See next comment.
Btw, if that would actually be the case, you would see that pretty quickly just by starting the app ;)